src/index.js

```diff
@@ -0,0 +1,158 @@
+import './style.css';
+import Firsticon from './images/refresh-cw.svg';
+import Secondicon from './images/corner-down-left.svg';
+import Thirdicon from './images/more-vertical.svg';
+import Fourthicon from './images/trash-2.svg';
+import {
+  toDoList, refreshDiv, enterDiv, refreshImg, enterImg, listSort, addInput, listWindow, clearAll,
+} from './modules/variables.js';
```

Comment:
Nice work by using ES6 modules to keep your functions in a separate file but I feel it would be better to put all those functions in a static class and then export the class, by doing so you can be able to call the individual function by using a dot operand of the class followed by the name of the function `class.function`.

Reply:
Thank you Victor. I've moved my class constructor for the list functions to the module that this import refers to. So all the functions that require those variables now run in one module